### ISSUE TITLE
Output of the title of the backend layout instead of the id

### ISSUE THREAD

**Issue by daUhradone** [TYPO3-Headless/headless#707]
**Describe the bug**
In all  docs it is descirbed and it would be expected that the name of the backend layout is used for the api, but the id is submitted

**To Reproduce**
Setup a backend layout and check the api
default is submitted as: -1

**Expected behavior**
title: string

**TYPO3 version and TYPO3 Headless version**
TYPO3 12 - headless latest

**Comment by twoldanski** [comment]
Yeah, I confirm this behavior. But it is not ext:headless bug per se, but inconsistent values from core, if you set layout via tsconfig files, you will get nice name (if you set one), but if you add record you will receive ID of record to handle on your side (in fluid context).

**Comment by twoldanski** [comment]
@lukaszuznanski we have 2 options to match core behavior or add consistent approach in headless.

**Comment by twoldanski** [comment]
@daUhradone what exactly is not working? Because from what I checked everything is working as expected (we match core behavior), only if you are using backend layouts as records instead of files (files in my opinion are much better solution) you have add extra step (prepare custom Typoscript) to match records to layouts same way you are doing in fluid context.
